package.json

```diff
@@ -64,6 +61,7 @@
     "fresh-require": "^1.0.3",
     "log": "^1.4.0",
     "loglevel": "^1.6.1",
+    "pino-pretty": "^1.0.0",
```

Comment:
I think you can remove chalk from dependencies.

Reply:
Oops. Fixed.

Reply:
Ditto for `split2`.  But it still has to be a dev dependency, I think.

Reply:
Thanks. It's moved.